modules/logging-release-notes-5-6-27.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-27_{context}"]
+= Logging 5.6.27
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10988[RHBA-2024:10988].
+
+[id="logging-release-notes-5-6-27-bug-fixes_{context}"]
+== Bug fixes
+None.
+
+[id="logging-release-notes-5-6-27-CVEs_{context}"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2018-12699[CVE-2018-12699]
+* link:https://access.redhat.com/security/cve/CVE-2019-12900[CVE-2019-12900]* link:https://access.redhat.com/security/cve/CVE-2024-9287[CVE-2024-9287]
```

Comment:
I think you've accidentally merged two lines.

Reply:
AH, yep. Fixed. TY!